### PR TITLE
Update my_shlex to Python 3.6 version

### PR DESCRIPTION
**Pull request by evanunderscore**

### CODE REVIEW

**Comment by evanunderscore** [comment]
@kislyuk Are you happy with removing this from the coverage report? Most of the code has nothing to do with us.

**Comment by kislyuk** [comment]
That's fine by me, yes.

File: argcomplete/my_shlex.py [hunk 1]
```diff
@@ -31,7 +33,9 @@
```
Comment (kislyuk) [inline argcomplete/my_shlex.py]: `split` is no longer defined, please remove this line altogether.

Comment (evanunderscore) [inline argcomplete/my_shlex.py]: Done, #194

**Pull request merged**